package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "ontotext-yasgui-web-component",
```

Comment:
Didn't we decided to name it `ontotext-yasgui` and the inner ontotext-yasgui to be `ontotext-yasgui-web-component`.